Dockerfile.dev

```diff
@@ -0,0 +1,55 @@
+# Build image
```

Comment:
What is this file for?

Reply:
If you don't have any importance/relying on `rust` being the base image, we can delete this

Reply:
Oh, so that you can still use it if you depends on `rust` being base image.

Reply:
Where would it be used? The images we use in self-hosted, and saas all come from the standard `Dockerfile`. Could we remove this dockerfile as it won't be used?